src/ios/SAMKeychain/SAMKeychain.h

```diff
@@ -151,11 +184,16 @@ extern NSString *const kSSKeychainWhereKey;
  @param accessibilityType One of the "Keychain Item Accessibility Constants"
  used for determining when a keychain item should be readable.
 
- If the value is `NULL` (the default), the Keychain default will be used.
+ If the value is `NULL` (the default), the Keychain default will be used which
+ is highly insecure. You really should use at least `kSecAttrAccessibleAfterFirstUnlock`
+ for background applications or `kSecAttrAccessibleWhenUnlocked` for all
+ other applications.
 
  @see accessibilityType
  */
 + (void)setAccessibilityType:(CFTypeRef)accessibilityType;
 #endif
 
 @end
+
+#import "SAMKeychainQuery.h"
```

Comment:
Unfortunately this is the only manual change required in the library. It previously read `#import <SAMKeychain/SAMKeychainQuery.h>` but I didn't know of a way to import these files as part of this framework.. Without having worked in iOS code for a few years now I would think this change could almost be submitted to SAMKeychain so that the reference is relative.. or we could figure out a way of making the cordova plugin recognize this framework